src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -41,6 +49,16 @@ const OwnerPanel = (props: Props) => {
     return 100;
   };
 
```

Comment:
Hmm I would prob just pass in seller since it's only going down one level. But if we end up creating new components and need to pass the props down more layers, then we should def use context